wp-directives.php

```diff
@@ -45,6 +45,7 @@ function () {
 require_once __DIR__ . '/src/directives/attributes/wp-context.php';
 require_once __DIR__ . '/src/directives/attributes/wp-class.php';
```

Comment:
I think the `require_once` for the `wp-html` directive is missing. 🤔 
```suggestion
require_once __DIR__ . '/src/directives/attributes/wp-class.php';
require_once __DIR__ . '/src/directives/attributes/wp-html.php';
```

Reply:
There is another `require_once` (line 38) that could cause some confusion, but it is not for the directive. ☝️

Reply:
Oh, nevermind. I just saw that it was already added. 👍